sample/redo/src/App.tsx

```diff
@@ -1,15 +1,74 @@
-import React from 'react';
+import React, {useEffect, useState} from 'react';
 import './App.css';
 
-import Replicache from 'replicache';
+import Replicache, {REPMHTTPInvoker, ReadTransaction} from 'replicache';
+import {diffServerURL, diffServerAuth, batchURL} from './settings';
+
+const repmInvoke = new REPMHTTPInvoker('http://localhost:7002').invoke;
+
+const rep = new Replicache({
+  diffServerURL: diffServerURL,
+  dataLayerAuth: '1',
+  diffServerAuth: diffServerAuth,
+  batchURL: batchURL,
+  repmInvoke,
+});
+
+registerMutations(rep);
 
 function App() {
-  console.log(Replicache);
+  const [allTodos, setAllTodos] = useState<Todo[]>([]);
+  // const [selectedListId, setSelectedListId] = useState<number>(-1);
+
+  // This is not the right way to do this. I just wanted something to show up on
```

Comment:
Huh. I would have thought just calling `setAllTodos` (e.g., without `useEffect` would have done it. What am I missing?).